options/train/BasicVSR/train_BasicVSR_GAN_C64N15L5_201600kft_pix1e-2per1GAN5e-3_B4G4.yml

```diff
@@ -0,0 +1,142 @@
+# general settings
+name: 1001_ReGAN_C64N15L5_201600kft_pix1e-2per1GAN5e-3_B4G4
+model_type: VideoRecurrentGANModel
+scale: 4
+num_gpu: 8  # set num_gpu: 0 for cpu mode
+manual_seed: 0
+
+# dataset and data loader settings
+datasets:
+  train:
+    name: REDS
+    type: REDSRecurrentDataset
+    dataroot_gt: datasets/REDS/train_sharp_with_val.lmdb
+    dataroot_lq: datasets/REDS/train_sharp_bicubic_with_val.lmdb
+    dataroot_flow: ~
+    meta_info_file: basicsr/data/meta_info/meta_info_REDS_GT.txt
+    val_partition: REDS4  # set to 'official' when use the official validation partition
+    test_mode: False
+    io_backend:
+      type: lmdb
+
+    task: VSR
+    num_frame: 15
+    num_interpolate: 0
+    gt_size: 256
+    interval_list: [1]
+    random_reverse: false
+    use_flip: true
+    use_rot: true
+
+    # data loader
+    use_shuffle: true
+    num_worker_per_gpu: 6
+    batch_size_per_gpu: 4
+    dataset_enlarge_ratio: 200
+    prefetch_mode: ~
+
+  val:
+    name: REDS4
+    type: VideoRecurrentTestDataset
+    dataroot_gt: datasets/REDS4/GT
+    dataroot_lq: datasets/REDS4/sharp_bicubic
+
+    cache_data: true
+    io_backend:
+      type: disk
+
+    num_frame: -1  # For recurrent, num_frame has no effect. Set to -1
+
+# network structures
+network_g:
+  type: BasicVSR
+  num_feat: 64
+  num_block: 30
+  spynet_path: experiments/pretrained_models/spynet_pretrained.pth
+
+network_d:
+  type: VGGStyleDiscriminator256
+  num_in_ch: 3
+  num_feat: 32
+
+# path
+path:
+  pretrain_network_g: experiments/pretrained_models/BasicVSR_REDS4.pth
+  strict_load_g: true
+  resume_state: experiments/1001_ReGAN_C64N15L5_201600kft_pix1e-2per1GAN5e-3_B4G4_archived_20210729_102126/training_states/10000.state
```

Comment:
In the public version , there should be no resume_state. 

Note: Not all your local changes should be pushed into GitHub